interpreter/binary/decode.ml

```diff
@@ -199,6 +199,11 @@ let memop s =
   let offset = vu32 s in
   Int32.to_int align, offset
 
+let simd_lane_memop s =
```

Comment:
I'd suggest inlining this, since it does not represent a standalone syntax object, and this factoring does not match the encoder.